test/unit/comment_test.rb

```diff
@@ -16,7 +16,12 @@ class CommentTest < ActiveSupport::TestCase
       assert comments.length > 0
     end
   end
-
+  test 'should have gmail quote' do
```

Comment:
Add after this line : 
require 'mail'
require 'nokogiri'

Reply:
Oh I didn't know that... Thanks for pointing that out!